packages/@lwc/shared/src/aria.ts

```diff
@@ -78,7 +78,7 @@ export type AccessibleElementProperties = {
     [prop in (typeof AriaPropertyNames)[number]]: string | null;
 };
 
-const { AriaAttrNameToPropNameMap, AriaPropNameToAttrNameMap } = /*@__PURE__*/ (() => {
+const { AriaAttrNameToPropNameMap, AriaPropNameToAttrNameMap } = /**@__PURE__*/ (() => {
```

Comment:
I'm not sure that these "pure" annotations work across all bundlers (Rollup, Webpack, etc.) when using the `/**` format rather than `/*`. Do we need to modify these?

(A cursory review, e.g. [for Webpack](https://webpack.js.org/guides/tree-shaking/#mark-a-function-call-as-side-effect-free) and [Rollup](https://rollupjs.org/configuration-options/#pure)), only mentions the `/*` format.)

Reply:
🤔 sometimes we use `/*@__PURE__*/` and sometimes we use `/*@__PURE__@*/`. What's up with that?

Reply:
Based on Rollup's docs, it looks like Rollup only checks for a substring, so it works either way.

To reduce confusion, I'd say we should use `/*#__PURE__*/` everywhere since this is what both the Webpack and Rollup docs say will work.

Reply:
FWIW, we do actually have a test (`verify-treeshakeable.js`) which will fail if we mess up the `PURE` annotations, but this is only for Rollup and only for certain modules.